crates/torii/indexer/src/processors/event_message.rs

```diff
@@ -28,6 +32,23 @@ where
         true
     }
 
+    fn task_priority(&self) -> usize {
+        1
+    }
+
+    fn task_identifier(&self, event: &Event) -> u64 {
```

Comment:
```suggestion
    fn task_identifier(&self, event: &Event) -> TaskId {
```
Same, only this one marked to be changed, but more are missing.